README.md

```diff
@@ -505,7 +558,7 @@ if (utils.isValidFragment(fragment)) {
 }
 ```
 
-Note that in the example above, using `utils.isValidFragment` might be unnecessary. It's possible to use directly `ValidTokenRegistryDataV2.guard` over the data.
+>**Note:** In the example above, it may be necessary to use `utils.isValidFragment`. It's possible to use `ValidTokenRegistryDataV2.guard` directly over the data.
```

Comment:
```suggestion
>**Note:** In the example above, it may be unnecessary to use `utils.isValidFragment`, as it's possible to use `ValidTokenRegistryDataV2.guard` directly over the data.
```